runner/src/server/services/cacheService.ts

```diff
@@ -91,12 +91,19 @@ export class CacheService {
 
     const initialisedSession = await this.cache.get(this.JWTKey(jwt));
 
-    const currentSession = await this.cache.get(userSessionKey);
-    const mergedSession = {
-      ...currentSession,
-      ...initialisedSession,
-    };
-    this.cache.set(userSessionKey, mergedSession, sessionTimeout);
+    if (config.overwriteInitialisedSession) {
+      request.logger.error("Replacing user session with initialisedSession");
```

Comment:
Can we make these logs `info` or `debug`?
We don't want to log `error` in the expected case.

Reply:
Ah thanks - good spot, made them errors during debugging to find them. Reverted to info.